docs/guide/samples-howto.md

```diff
@@ -1,7 +1,7 @@
 ## How To
 
 This section gives various examples of complex behaviors and/or functionalities.
-The examples are located in "[WEBOTS\_HOME/projects/samples/howto/]({{ url.github_tree }}/projects/samples/howto/)". Each example has his own controllers and worlds directory.
+The examples are located in "[WEBOTS\_HOME/projects/samples/howto/]({{ url.github_tree }}/projects/samples/howto/)". Each example has its own controllers and worlds directories.
```

Comment:
Just a side note (no need to change it now): in the documentation we write each sentence on a new line.